nemo/collections/multimodal/speech_llm/data/lhotse_s2s_dataset.py

```diff
@@ -135,13 +239,30 @@ def __getitem__duplex_(self, cuts) -> dict[str, torch.Tensor | list[str] | dict]
                     raise Exception("First speaker should be user")
 
                 if supervisions[1].speaker == "agent":
+                    use_word_alignment = getattr(cut, "s2s_duplex_align", False)
+                    text = supervisions[1].text
                     pattern = r"<\|\d+\|>"
-                    output_text = re.sub(pattern, "", supervisions[1].text)
+                    if not use_word_alignment:
```

Comment:
can u do a regression test with not use_word_alignment and train for 1 validation interval to see no regression on bleu

Reply:
I did a side-by-side comparison. text bleu is similar: https://wandb.ai/aiapps/salm_s2s_huk/reports/Regression-test-for-PR-11652--VmlldzoxMDY2NDQwOA